new-site/src/components/ComponentsList.scss

```diff
@@ -0,0 +1,18 @@
+@import '~hds-design-tokens/lib/all.scss';
```

Comment:
Can you run the [linter](https://github.com/City-of-Helsinki/helsinki-design-system/pull/726)? IIRC, it requires double quotes in CSS.

Reply:
- [ ] Use double quotes